src/quadrature/Ordinate_Space.hh

```diff
@@ -215,13 +215,11 @@ public:
   virtual vector<double> M() const = 0;
 
   //! Should the moment space be pruned to the specified order?
-  // virtual bool prune() const {
-  //   return true;
-  //   // By default, prune any moments beyond the user-specified expansion
-  //   // order. Such moments are included in Galerkin methods for purposes of
-  //   // computing the M and D matrices, but are then removed from the moment
-  //   // space unless the GQF interpolation model has been specified.
-  // }
+  virtual bool prune() const {
+    return true;
+    // By default, prune any moments beyond the user-specified expansion order.
+    // Overridden by Galerkin_Ordinate_Space::prune().
+  }
```

Comment:
There is no test for this bit of new code.  Can you generate something to cover this case?

Ref: https://codecov.io/gh/lanl/Draco/compare/a118ab1cbbd7a27757cd6bc99e04a81aa1d6abd7...7d9dc881f45c58711e06085b5975d8f8736d265c/diff

Reply:
Fixed by 0361632